src/main/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -170,14 +170,20 @@
             mutex = {"BARCODE_PARAMS", "LIBRARY_PARAMS"})
     public String LIBRARY_NAME;
 
-    @Argument(doc = "The name of the sequencing center that produced the reads.  Used to set the RG.CN tag.", optional = true)
-    public String SEQUENCING_CENTER = "BI";
+    @Argument(doc = "The name of the sequencing center that produced the reads.  Used to set the @RG->CN header tag.")
```

Comment:
We will need to double check that we aren't relying on this being optional in any of our workflows

Reply:
We are...this needs to change as otherwise external users of IBCTS are inadvertently generated uBAMs that say "CN:BI" (meaning that they were sequenced at Broad).....

Reply:
I agree that this should probably mean a minor version bump to picard though.